airgun/views/rhai.py

```diff
@@ -10,7 +10,30 @@
 )
 
 from airgun.views.common import BaseLoggedInView
-from airgun.widgets import ActionsDropdown, SatTable
+from airgun.widgets import (
+    ActionsDropdown,
+    SatTable,
+    GenericRemovableWidgetItem,
+    RemovableWidgetsItemsListView,
+)
+
+
+class InsightsOrganizationErrorView(BaseLoggedInView):
+    """View displayed when no Organization is selected or when the current organization has no
+    manifest
+    """
+    title = Text("//article[@id='content']/section/h1")
+    message = Text("//article[@id='content']/section/p")
+
+    @property
+    def is_displayed(self):
+        return (self.browser.wait_for_element(self.title, exception=False) is not None
+                and self.browser.wait_for_element(self.message, exception=False) is not None)
+
+    def read(self, widget_names=None):
+        if widget_names:
```

Comment:
as far as I understand, we know that `if not self.view.is_displayed:`  and we know that no widgets are displayed, so I suggest to use plain:
`return '{0}: {1}'.format(self.title.read(), self.message.read())`